src/util.ts

```diff
@@ -280,3 +297,36 @@ export const addHashToBaseline = (hash: string, filepath: string): void => {
     ignoreMessages: baselineErrorsFile.meta.ignoreMessages
   })
 }
+
+export const toGitLabOutputFormat = (
+  errorSummaryMap: ErrorSummaryMap,
+  specificErrorMap: SpecificErrorsMap,
+  errorOptions: ErrorOptions
+): string => {
+  const result: GitLabErrorFormat[] = []
+
+  for (const [key, error] of errorSummaryMap.entries()) {
+    const specificErrors = getSpecificErrorsMatchingSummary(
+      error,
+      specificErrorMap,
+      errorOptions
+    )
+
+    const specificError: SpecificError = specificErrors[0] || {}
```

Comment:
This might be a stupid question, but why are processing only the FIRST and not ALL of the `specificErrors` just like the human output does? 
If an error would occur multiple times in a file, it should be layed out so in the report in Gitlab too.

And another one: If there is really a reason for only using the first element, we should probably handle it not being there somehow. The fallback to an empty object should cause the script for fail. Or is there a valid reason for this fallback behavior to occur. Just asking, because I do not see that fallback in the `human readable` output.

Reply:
I now iterate through the specific errors and append the index to the fingerprint hash.